### ISSUE TITLE
A11Y known issues

### ISSUE THREAD

**Issue by Zaexides** [Zaexides/DutchTubers#1]
- GitHub icon does not have ALT text.
- Navigation menu items does not have proper contrast.
- Live indicator does not have proper contrast.
